src/ui/widgets/GroupingContainer/groupingContainer.tsx

```diff
@@ -1,29 +1,33 @@
-import React from "react";
+import React, { CSSProperties } from "react";
 
 import { Widget } from "../widget";
 import { WidgetPropType } from "../widgetProps";
 import { registerWidget } from "../register";
-import { StringProp, ChildrenPropOpt, InferWidgetProps } from "../propTypes";
+import {
+  StringProp,
+  ChildrenPropOpt,
+  InferWidgetProps,
+  BorderPropOpt
+} from "../propTypes";
 
 const GroupingContainerProps = {
   name: StringProp,
-  children: ChildrenPropOpt
+  children: ChildrenPropOpt,
+  border: BorderPropOpt
 };
 
 // Generic display widget to put other things inside
 export const GroupingContainerComponent = (
   props: InferWidgetProps<typeof GroupingContainerProps>
-): JSX.Element => (
-  // Uses an inner margin for children similar to Phoebus
-  // This prevents the title being overwritten
-  // Could be changed or perhaps customisable as a prop
-  <fieldset>
-    <legend>{props.name}</legend>
-    <div style={{ position: "relative", height: "100%", width: "100%" }}>
-      {props.children}
-    </div>
-  </fieldset>
-);
+): JSX.Element => {
+  const style: CSSProperties = {
+    position: "relative",
+    height: "100%",
+    width: "100%",
+    ...props.border?.css()
+  };
+  return <div style={style}>{props.children}</div>;
+};
```

Comment:
This is basically just a box now? Could the same thing be achieved by using `Display` to do this and leave this one with the `fieldset` tag? It was intended to replace the CS-Studio grouping container

Reply:
This is actually a little bit awkward. Although the default look of a grouping container has a group box, this should really be part of the border style rather than the grouping container widget. Other widgets can have a group box style too. However, it is different to other styles of border in that it requires additional elements. 

Many groups in CS-Studio and EDM don't want the border and it messes up the layout if you add it.

As well as all this, the behaviour in bob files is a bit unusual - if you remove the group box style border it changes the x and y coordinates of the contents to compensate.

It needs a bit of thought, but I think this way round is cleaner for the time being.

Reply:
So in CS-Studio you can have a group box that doesn't have the heading and border?

Reply:
Yes, and we often do.

Reply:
I didn't know that 😮

But if you *did* have a group box with a label in CS-Studio, it wouldn't look right here?

Reply:
Right. But this is actually less common for us.

Reply:
Okay, in that case, could you map the CS-Studio "grouping container" to the "Display" widget?

I think that they end up being very similar, but the `GroupingContainer` would still be a different, separate thing that we could make use of.